task/generate-olm-bundle/0.1/generate-olm-bundle.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: generate-olm-bundle
+spec:
+  description: Generate an OLM bundle
+  params:
+    - name: input-dir
+      description: Directory to read cluster-ready operator manifests from
+      default: deploy
+    - name: channels
+      description: Comma-separated list of channels the bundle belongs to
+      default: alpha
+    - name: version
+      description: Semantic version of the operator in the generated bundle
+    - name: package-name
+      description: Bundle's package name
+  workspaces:
+    - name: source
+      description: Workspace with the source code
+  steps:
+    - name: generate-olm-bundle
+      # TODO: ensure this image is built within konflux
```

Comment:
What are you trying to ensure is built? The operator-sdk image?

Reply:
Yes! I haven't found the pipeline files for this so I'm just going off of what Joe said.

Reply:
Do we still need this TODO?